strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/maven/MavenArtifactControllerTest.java

```diff
@@ -841,15 +841,39 @@ public void testThatMavenProxyRepoDoesNotBlockTheConnectionPool()
                .get(url)
                .peek()
                .then()
-               .statusCode(HttpStatus.NOT_FOUND.value());
+               .statusCode(HttpStatus.OK.value());
+    }
+
+    @Test
+    public void shouldDownloadProxiedSnapshotArtifact()
+            throws Exception
+    {
+        String url = getContextBaseUrl() +
```

Comment:
In time, this test will not be reproducible. The timestamps for snapshot artifacts graudally move on, and artifact repository managers remove older timestamps, which will at some point make this URL not work. For the time-being, this test will work, but at some point unexpectedly, it will start failing.